src/Components/App/App.scss

```diff
@@ -1,20 +1,13 @@
 * {
   padding: 0;
   margin: 0;
-};
-
-// button {
-//   border-radius: 10%;
-//   color: #f3f5f6;
-//   height: 3vh;
-//   border: solid 1px #121316;
-//   background-color: #121316;
-//   width: 20vw;
-//   margin: 1%;
-// }
-
-:focus{
-  outline: none;
+  font-family: 'Abel', sans-serif;
```

Comment:
Glad to see we got these fonts implimented, do really feel like it makes a difference.